tests/sentry/incidents/endpoints/test_serializers.py

```diff
@@ -151,6 +151,22 @@ def test_environment(self):
         serializer.save()
         assert len(AlertRuleEnvironment.objects.filter(alert_rule=alert_rule)) == 0
 
+    def test_environment_non_list(self):
+        base_params = self.valid_params.copy()
+        env_1 = Environment.objects.create(organization_id=self.organization.id, name="test_env_1")
+        Environment.objects.create(organization_id=self.organization.id, name="test_env_2")
```

Comment:
Why did you create this?

Reply:
Copy/pasted from the other test, and vaguely thought it'd be helpful to have another env around, but maybe it's not useful. I'll tidy up all the code around this once I remove the shim.